src/server/document-controller.spec.js

```diff
@@ -5,6 +5,7 @@ import uniqueId from '../utils/unique-id.js';
 import { assert, createSandbox } from 'sinon';
 import DocumentController from './document-controller.js';
 import { afterEach, beforeEach, describe, expect, it } from 'vitest';
+import { ROLE } from '../domain/constants.js';
```

Comment:
This line is very short ;-)

Reply:
And as they say, the short ones should go first 😆